app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'author_id'
+
+  def recent_post
+    posts.limit(3).order(created_at: :desc)
```

Comment:
- [ Required ] I think this query should first select all the posts from the database, sort them in descending order, and finally limit them to only three. I think you could refactor it as shown below, the same way you did it in the `Post` model for the `most_recent_comments` method.

```ruby
  posts.order(created_at: :desc).limit(3)
```